packages/flutter/lib/src/gestures/mouse_tracking.dart

```diff
@@ -69,7 +58,7 @@ class MouseTrackerAnnotation {
   ///  * A pointer has moved onto, or moved within an annotation without buttons
   ///    pressed.
   ///
-  /// This callback is not triggered when
+  /// This callback is not triggered when:
```

Comment:
(sidenote): Now that we are working on providing local coordinates in the events this should presumably change?

Reply:
Since we're delaying the local coordinates it's not gonna be soon. 
Additionally, I'm still unsure if we should dispatch hover when the widget moves, because it might be expensive during animations.